spec/decorators/odk/form_decorator_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+# Form decorator spec
+module Odk
+  describe FormDecorator, :odk, :reset_factory_sequences do
+    describe "needs_manifest?", :odk do
+      context "for form with single level option sets only" do
+        before { @form = create(:form, question_types: %w(select_one)) }
+        it "should return false" do
+          decorated_form = decorate(@form)
```

Comment:
Notice the duplication here and below. That is a big sign to use a `let` block in the parent context. Please refactor.